performance/ring/src/main/kotlin/org/jetbrains/ring/SingletonInitBenchmark.kt

```diff
@@ -0,0 +1,18117 @@
+/*
```

Comment:
This file is actually generated by https://github.com/google/googletest/blob/master/googlemock/scripts/pump.py